### DISCUSSION TITLE
新帖好少

### DISCUSSION THREAD

**Discussion by ArcSurge** [share121/inter-knot#2575] [其他]
稍微向下一翻就是几天以前的

**Comment by Bai6869** [comment]
完全同意 活力不够

**Comment by guguxuyao648** [comment]
绳网嘛，没点特殊手段进不来的，帖子少很正常

**Comment by iuhjk** [comment]
挺正常的，倒不如说每天固定有新帖子才让我惊讶<img src="https://img-static.mihoyo.com/communityweb/upload/4a102d21856350263e83278ac8c48526.png" alt="吃惊" width="75" >

**Comment by fashibuding1214** [comment]
而且说实话，咱也不知道要在这发什么<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Reply by Goodname-LOL** [reply]
+1

**Comment by jinitaimei000** [comment]
表情为啥发不出去啊<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8a1c5e858456b720f2c3c45982729628_6809351812774032443.gif" alt="米游姬 干杯" width="75" >

**Comment by I-ikun** [comment]
确实

**Comment by lastbeyond** [comment]
现在人太少了<img src="https://bbs-static.miyoushe.com/static/2024/06/28/da03838dd95a950d834f2fbb7fb75258_6545789758486199655.png" alt="「灯」-穷" width="75" >
